operator/src/main/java/oracle/kubernetes/operator/Watcher.java

```diff
@@ -226,12 +227,16 @@ private long computeNextResourceVersionFromMessage(V1Status status) {
         if (index2 > 0) {
           String val = message.substring(index1 + 1, index2);
           if (!isNullOrEmpty(val)) {
-            return Long.parseLong(val);
+            try {
+              return new BigInteger(val);
+            } catch (NumberFormatException nfe) {
```

Comment:
Long.parseLong can throw NumberFormatException - why would you swallow it for BigInteger?